src/main/java/dao/EventDAO.java

```diff
@@ -2,6 +2,10 @@
 
 import main.java.models.Event;
 
+import java.util.List;
+
 public interface EventDAO extends CRUD<Event> {
 
+    List<Event> getEventList() throws DAOException;
```

Comment:
Maybe this should be `getList()` in case we add it into the CRUD in the future.

Reply:
Done!